automation/test.sh

```diff
@@ -452,7 +449,7 @@ fi
 
 
 # Run functional tests
-FUNC_TEST_ARGS=$ginko_params make functest
+FUNC_TEST_ARGS=$ginko_params FUNC_TEST_LABEL_FILTER=$label_filter make functest
```

Comment:
```suggestion
FUNC_TEST_ARGS=$ginko_params FUNC_TEST_LABEL_FILTER="--label-filter=(${label_filter})" make functest
```
That would allow cleaning up the rest of the file to make it more readable. WDYT?

Reply:
Thanks @jean-edouard for the suggestion. For sure this decrease the number of `--label-filter=` I wrote :smile: 
Done! PTAL

Reply:
I also rebased on main, once the tests are passed I'll re-do the comparison with other PR to check if there are some new tests that should be addressed. Thanks